.github/workflows/jit.yml

```diff
@@ -57,9 +57,10 @@ jobs:
       fail-fast: false
       matrix:
         target:
-          - i686-pc-windows-msvc/msvc
-          - x86_64-pc-windows-msvc/msvc
-          - aarch64-pc-windows-msvc/msvc
+# To re-enable later when we support these.
```

Comment:
Can we add a link in the comments to the issue where we're tracking this?

Reply:
Mark says he will add normal switch-case support back to this PR by building on top of it I think. So we should get it nearly immediately after this lands.